Modules/_testcapimodule.c

```diff
@@ -1205,7 +1205,8 @@ test_k_code(PyObject *self)
     value = PyLong_AsUnsignedLongMask(num);
     if (value != (unsigned long)-0x42)
         return raiseTestError("test_k_code",
-            "PyLong_AsUnsignedLongMask() returned wrong value for long 0xFFF...FFF");
+                              "PyLong_AsUnsignedLongMask() returned wrong "
+                              "value for long -0xFFF..000042");
```

Comment:
first we have:
`num = PyLong_FromString("-FFFFFFFF000000000000000042", NULL, 16);`
and then the check is whether PyLong_AsUnsignedLongMask returned -0x42,
i.e. truncated num as expected.
so yes, IMHO -0xFFF..000042 is the right value.

and I guess the `..` means a big enough amount of zeros, such that the
truncation would indeed result in -0x42.
ISTM the original author felt this would be more elegant than writing
`-0xFFFFFFFF000000000000000042` again...